src/GraphQL/ResolveFieldContext/ResolveFieldContextExtensions.cs

```diff
@@ -61,7 +61,7 @@ private static bool TryGetArgument(this IResolveFieldContext context, Type argum
         /// <summary>Determines if the specified field argument has been provided in the GraphQL query request</summary>
         public static bool HasArgument(this IResolveFieldContext context, string name)
         {
-            var isIntrospection = context.ParentType == null ? context.FieldDefinition.IsIntrospectionField() : context.ParentType.IsIntrospectionType();
+            var isIntrospection = context.ParentType?.IsIntrospectionType() ?? context.FieldDefinition.IsIntrospectionField();
```

Comment:
This expression is not equivalent to the previous one.

Reply:
> @srs6814 I reviewed almost all the files and only when I came to the very end I noticed that your PR is targeting the `master`. We are preparing a v4 release and all the work is happening in the `develop` branch. Please change your target branch.

Agreed.

Reply:
@sungam3r 
isn't it similar to the previous one?

if context.ParentType == null, it fallbacks to context.FieldDefinition.IsIntrospectionField

Reply:
`context.ParentType` may be not null, but `context.ParentType.IsIntrospectionType()` still _may be_ null. There is no fallback in the  first case.